demo/app-root.ts

```diff
@@ -32,6 +32,8 @@ export class AppRoot extends LitElement {
 
   @state() private colGap: number = 1.7;
 
+  @state() private loggedIn: boolean = false;
```

Comment:
can we be consistent on naming booleans? would prefer this to be `isLoggedIn` as on some places we also use `is` for naming

Reply:
I prefer to use the `is...` prefix where it fits. The "In" in "loggedIn" serves as an implicit "is"
logged in -> true | false
"is logged in" is redundant, less clear, and not ... good English

"is required" is clearer as true | false than "required"

Reply:
This is different in offshoot, because we're passing in the entire user object
https://git.archive.org/www/offshoot/-/merge_requests/216/diffs